pom.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.2.7.RELEASE</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>uk.ac.ebi.eva</groupId>
+	<artifactId>contig-alias</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>contig-alias</name>
+	<description>Service to provide synonyms of chromosome/contig identifiers </description>
+
+	<properties>
+		<java.version>11</java.version>
```

Comment:
Just to let you know, most of our projects are still on java 8, but I think it's a good idea to start trying projects on java 11.